src/trace/trace.c

```diff
@@ -113,9 +113,22 @@ bool trace_filter_update(uint32_t uuid_addr, int trace_level)
 	return -EINVAL;
 }
 
+/**
+ * \brief Runtime trace filtration
+ * \param lvl log level (error, info, debug ...)
+ * \param uuid uid address
```

Comment:
filtering would be more be appropriate here

Reply:
changed